lib/src/bloc/movies_bloc.dart

```diff
@@ -19,8 +19,28 @@ class MoviesBloc extends IMoviesBloc {
   @override
   void fetchMovies() async {
     final movieList = await _moviesRepository.fetchAllMovies();
+    _sinkMovies(
+      movieList,
+    );
+  }
+
+  @override
+  void fetchMovieSearch(String movieKeywords) async {
+    if (movieKeywords != '') {
```

Comment:
Here you can use a method of Strings call isNotEmpty

Reply:
Done